aws_promise/s3bucket.js

```diff
@@ -76,6 +76,106 @@ module.exports = {
       params.ACL = input.acl;
     }
     return s3.putObject(params).promise();
-  }
+  },
+
+  addBucketPolicy: function(input) {
+    var s3 = this.findService(input);
+    var policy = {
+        "Id": "PolicyID-"+input.uid+"",
```

Comment:
Can we have the static part of the policy in a configuration file

Reply:
This is same as we have in BitBucket, and it's optional.